packages/odyssey-ie-11-sandbox/src/ErrorBoundary.scss

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* stylelint-disable */
```

Comment:
it would be better to selectively disable specific rules here rather than full stop preventing all static analysis! What are you trying to accomplish that stylelint doesn't like?

Reply:
resolved in 7a4d915